src/AdminAcceptanceTests.Tests/Features/OrganisationDashboard/OrganisationsUserAccountsDashboard.feature

```diff
@@ -6,15 +6,15 @@
 Background:
 	Given that an Authority User has logged in on Public Browse
 	And an Authority User clicks the admin tile on the Public Browse homepage
-
+@SmokeTest
```

Comment:
might as well put it on the feature, the last ignored scenario seems harmless when its working

Reply:
good point, see next commit